Citygis wfs & Twitter func

bartverhaar opened the pull request.

bartverhaar committed:
Bugfix for incidents module with citygis_wfs.

bartverhaar committed:
- Dropdown with buttons not always showing
- Twitter func call
- Added authorized link to GeoServer for citygis_wfs

matthijsln commented on public/js/safetymaps/modules/incidents/IncidentMonitorController.js:
```diff
@@ -554,7 +554,12 @@ IncidentMonitorController.prototype.updateVehiclePositionLayerCityGISWFS = funct
+	$.ajax({
+        url: me.options.vehicleSourceURL,
+        beforeSend: function (xhr) {
+            xhr.setRequestHeader ("Authorization", "Basic " + me.options.vehicleSourceURLauth); //btoa("username:password")
```
De authenticatie van deze WFS moet niet naar de client. Deze wordt gereverse-proxied door Apache die de Authorization header toevoegt. Voor versie 5 met mod_pubtkt.

bartverhaar replied:
Removed client auth.

bartverhaar committed:
See comment on from matthijsln on Citygis wfs & Twitter func #17

De authenticatie van deze WFS moet niet naar de client. Deze wordt gereverse-proxied door Apache die de Authorization header toevoegt. Voor versie 5 met mod_pubtkt.

matthijsln reviewed (approved):
This PR contains 3 separate things, atm very small overseeable changes so not a problem. In case such a PR grows overtime separate features should have their own PR. To deploy a test version containing multiple PR's, create a new branch and merge the PR's in them, or base a PR on another PR if it depends on changes in it.
